doc/source/parameters/cfd/dynamic_flow_control.rst

```diff
@@ -1,39 +1,61 @@
+The purpose of this subsection is to enable dynamic flow control. It is important when we want to simulate an average
+velocity on a specific boundary (CFD) or the whole domain (for CFD-DEM). To control the average velocity of the flow, the code
+calculates a :math:`\beta` coefficient at each time step that is used to keep the average velocity in an acceptable
+threshold. If the chosen wall boundary on which a average velocity is imposed is an inlet, the average velocity
+targeted must be negative since the outward normal vector is used for the calculation. Thereby, if the chosen wall
+boundary is an outlet, average velocity must be positive. In other to keep the sign consistency, the targeted
+velocity should have an negative sign when used with CFD-DEM even if no normal vector is used since the
+whole domain is taken into account.
+
+The main controller of the average velocity is the following equation and is based on Wang [1]:
+
+.. math::
+    \beta^{n+1} = \beta^n - \frac{\alpha}{\Delta t} \left[ (\bar{U})^{0} - 2(\bar{U})^{n} + (\bar{U})^{n-1} \right]
 
 The default parameters are:
 
 .. code-block:: text
 
   subsection flow control
-    set enable = false
-    set volumetric flow rate = 0
-    set boundary id = 0
-    set flow direction = 0
-    set initial beta = 0
+    set enable                  = false
+    set enable beta particle    = false
+    set average velocity target = 0
+    set boundary id             = 0
+    set flow direction          = 0
+    set initial beta            = 0
+    set alpha                   = 1.0
+    set verbosity               = quiet
   end
 
-* The ``enable`` parameter is set to ``true`` to enable an imposed volumetric flow rate.
+* The ``enable`` parameter is set to ``true`` to enable an imposed average velocity.
 
-* The ``volumetric flow rate`` parameter specifies the flow rate (:math:`m^{dim}/s`). 
+* The ``enable beta particle`` parameter is set to ``true`` to apply a proportional beta force on particles.
+
+* The ``average velocity target`` parameter specifies the average velocity (:math:`m/s`). The value is compared to the calculated value at a boundary (for CFD solvers) or the whole domain (for CFD-DEM solver).
 
 .. warning::
 
-  This value should be negative if imposed on an the inlet boundary, and positive if imposed on an outlet boundary.
+  This value should be negative if imposed on an the inlet boundary (or CFD-DEM), and positive if imposed on an outlet boundary.
 
 .. note::
 
-  If you are simulating a problem with periodic boundary conditions at inlet and outlet walls, it is important to control the volumetric flow rate specifying the boundary id that corresponds to the inlet only (where the flow would have a negative value).
+  If you are simulating a problem with periodic boundary conditions at inlet and outlet walls, it is important to control the average velocity specifying the boundary id that corresponds to the inlet only (where the flow would have a negative value).
```

Comment:
```suggestion
  If you are simulating a problem with periodic boundary conditions, it is important to control the average velocity by specifying the boundary id that corresponds to the inlet only (where the flow would have a negative value).
```

Would it work if. you specify the outlet only? Why the inlet? isnt it just one of the boundary only and that's it?

Reply:
I don't remember very well why there's this comment. I think it would be better to use inlet only, so remove any mention of outlet boundary. Also, maybe I can remove the - sign for the normal vector, it makes more sense with the average velocity.